src/components/frame/useFrame.ts

```diff
@@ -39,7 +39,7 @@ export const useParentFrame = function ({
         })
         .catch((err) => {
           trace("connectToParent failed: ", err);
-          setStatus("DISCONNECTED");
+          setStatus("TIMEOUT");
```

Comment:
considering there may be other rejection, should we check the rejection is about timeout, and not something else ?